components/FederalBanner.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { css } from 'react-emotion'
+import { GoCSignature } from '@cdssnc/gcui'
+import Link from 'next/link'
+
+const fbanner = css`
+  background-color: white;
+  color: black;
+  padding-left: 60px;
+  padding-top: 10px;
+  padding-bottom: 3px;
+`
+const frlink = css`
+  text-align: right;
+  background-color: white;
+  color: black;
+  padding-right: 60px;
+`
+const container = css`
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
```

Comment:
Does this snap down to anything on mobile? This is a pretty simple component, so we should figure it out on something that's easy.